citi-onboarding/arte-que-salva-alma-front
# Feature(banner-secion): Adding initial implementation

Implementation of the banner

- in this implementation, the assets and text are not being consumed from the back-end. they're added straight to the code for the time being until the next pull request. 

## How to test?

1. Open the terminal and go to project folder

1. In terminal:
   1. `git checkout develop`
   1. `git pull`
   1. `git checkout feature-banner-section`
   1. `git pull origin feature-banner-section`
   1. `yarn install` ou `npm install`
   1. `code .`
   1. `yarn start` ou `npm start`

1. In a browser go to http://localhost:3000

1. Verify if the component or section looks like the mockup with different sizes of screens

1. Read section change files on GitHub

1. Test the things you think are necessary to test, even those that are not in this description
